.eslintignore

```diff
@@ -16,3 +16,5 @@ src/util/eventValidations.js
 **/trackingPlan*
 src/v0/destinations/personalize/scripts/
 test/integrations/destinations/testTypes.d.ts
+coverage/
```

Comment:
This is not required. This is reminiscent of the older eslint configuration. You can delete this directory now.